src/lib/core/CHIPVendorIdentifiers.hpp

```diff
@@ -34,10 +34,15 @@ namespace chip {
 // CHIP Vendor Identifiers (16 bits max)
 //
 
+// As per specifications (section 2.5.2 Vendor Identifier)
 enum VendorId : uint16_t
 {
     Common       = 0x0000u,
-    NestLabs     = 0x235Au,
+    GoogleNest   = 0x235Au,
```

Comment:
While you're here, want to drop this one?  Unless that's actually the right vendor id?

Reply:
@emargolis just updated this vendor ID in this PR. So likely this is correct.

Reply:
@emargolis updated the name, not the value.....  Anyway, we can leave that for a separate PR.